javascript/big-practice/src/scripts/utils/fetchAPI.js

```diff
@@ -1,6 +1,45 @@
 import { API_MESSAGE } from '../constants/message';
 
 const URL = 'http://localhost:3000/notes';
+const userURL = 'http://localhost:3000/users';
```

Comment:
With an API port like this, you should define a contact for it, so you can change the port if you want whenever.